bgw-net/spring-server/src/main/resources/application.properties

```diff
@@ -0,0 +1,7 @@
+spring.datasource.url = jdbc:postgresql://localhost:5432/myDB
```

Comment:
This should probably get removed since the datasource properties are a "local" configuration. Same goes for lines two and three.